forge-game/src/main/java/forge/game/keyword/Keyword.java

```diff
@@ -93,7 +93,7 @@ public enum Keyword {
     HASTE("Haste", SimpleKeyword.class, true, "This creature can attack and {T} as soon as it comes under your control."),
     HAUNT("Haunt", SimpleKeyword.class, false, "When this is put into a graveyard, exile it haunting target creature."),
     HEXPROOF("Hexproof", Hexproof.class, false, "This can't be the target of %s spells or abilities your opponents control."),
-    HIDEAWAY("Hideaway", SimpleKeyword.class, false, "This permanent enters the battlefield tapped. When it does, look at the top four cards of your library, exile one face down, then put the rest on the bottom of your library."),
+    HIDEAWAY("Hideaway", KeywordWithAmount.class, false, "When this permanent enters the battlefield, look at the top %d cards of your library, exile one face down, then put the rest on the bottom of your library."),
```

Comment:
Is it worth it to have the Lang.getNumeral in here, too?

Probably could do it in Card.java by moving it out of the list?

Reply:
i think i would done this as part of the keyword parsing so it knows there that it needs to turn the number into the word